_translator-files/po/de/Client-Troubleshooting.po

```diff
@@ -45,228 +47,230 @@ msgid ""
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:48
 msgid "Exactly how you avoid listening to your direct signal will depend on your individual setup - your sound interface, mixing desk, headphone connection point, etc. For example, some audio interfaces have \"monitor\" buttons (turn these off), or similar options. **If you are still having problems**, try asking on the [forum](https://github.com/jamulussoftware/jamulus/discussions)."
-msgstr ""
+msgstr "Wie genau du vermeidest, dass du dein direktes Signal hörst, hängt von deiner individuellen Konfiguration ab - von deinem Audio-Interface, Mischpult, Kopfhöreranschluss usw. Einige Audioschnittstellen haben zum Beispiel \"Monitor\"-Tasten (schalte diese aus) oder ähnliche Optionen. **Wenn du immer noch Probleme hast, versuche im [Forum](https://github.com/jamulussoftware/jamulus/discussions) zu fragen."
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:50
 msgid "Be aware that while listening to the Server's signal will ensure you will be in sync with other musicians, you may also experience problems if your overall latency (indicated by the \"Delay\" light in Jamulus) is not green or at least yellow most of the time. Consult the [software manual](/wiki/Software-Manual) to understand how to adjust your setup to help with this."
-msgstr ""
+msgstr "Beachte dass das Abhören des Server Signals zwar sicherstellt, dass du mit den anderen Musikern synchron bist, dass es aber auch zu Problemen kommen kann, wenn die Gesamtlatenz (angezeigt durch die \"Delay\"-Lampe in Jamulus) die meiste Zeit über nicht grün oder zumindest gelb ist. Lies das [Software-Handbuch](/wiki/Software-Manual), um zu erfahren, wie du dein Setup in diesem Fall anpassen kannst."
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:52
 msgid "### Can't work out your mic settings?"
-msgstr ""
+msgstr "### Du kannst deine Mikrofoneinstellungen nicht herausfinden?"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:54
 msgid "When using a microphone while playing your other instrument, you can use a stereo audio input signal in your settings where one channel is connected to the instrument and the other channel is connected to a microphone signal. On the microphone channel an optional reverberation effect can be applied."
-msgstr ""
+msgstr "Wenn du ein Mikrofon verwendest, während du ein anderes Instrument spielst, kannst du in deinen Einstellungen ein Stereo-Audioeingangssignal verwenden, bei dem ein Kanal mit dem Instrument und der andere Kanal mit einem Mikrofonsignal verbunden ist. Auf den Mikrofonkanal kann ein optionaler Hall-Effekt eingestellt werden."
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:56
 msgid "### Buffer LEDs suddenly going red, outages, jittering, weird sounds?"
-msgstr ""
+msgstr "### Puffer-LEDs leuchten plötzlich rot, Ausfälle, Jittering, seltsame Geräusche?"
```

Comment:
"jittering" sounds strange...

Reply:
Corrected changed to "zitternd" OK?

Reply:
Hmm. No. Sounds also a bit strange.

Reply:
Added link to Wikipedia explaining Jitter. Ok?